### PR TITLE
Add rewind

### PR DESCRIPTION
**Pull request by devonharvey**

### CODE REVIEW

**Comment by ardsouza** [comment]
lgtm

File: client/js/views/visualize.react.jsx [hunk 1]
```diff
@@ -92,15 +100,20 @@ var Visualize = React.createClass({
...
-    this.setState( {commitIndex: 0, currentPath: '', fileTree: fileTree, filePaths : {}} );
-    this.updatePaths();
+    this.state.filePaths = {};
+    this.updatePaths(0);
+    this.setState( {commitIndex: 0, currentPath: '', fileTree: fileTree, playbarDirection: 'forward'} );
   },
 
   getInitialState: function() {
-    return {loading: true, windowHeight: $(window).height() - 305, commits: [], commitIndex: 0, currentPath: '', fileTree: {}, filePaths : {}, showFileGitualize: false, urls: {form: '', to: ''}, help: 'Read up on tips and tricks!'};
+    return {loading: true, windowHeight: $(window).height() - 305, commits: [], commitIndex: 0, currentPath: '', fileTree: {}, filePaths : {}, playbarDirection: 'forward', showFileGitualize: false, urls: {form: '', to: ''}, help: 'Read up on tips and tricks!'};
```
Comment (dxuehu) [inline client/js/views/visualize.react.jsx]: pass help to modal as a property

File: client/js/utils/fileTreeUtils.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-module.exports.updateTree = function(currentCommit, fileTree) {
+
+module.exports.updateTree = function(currentCommit, fileTree, direction) {
   var filepath;
-  currentCommit.files.forEach(function(file) {
-    //filepath = file.filename;
-    file.status === 'removed' ? removeFile(fileTree, file) : addFile(fileTree, file);
-  });
+  if (direction === 'backward') {
+    currentCommit.files.forEach(function(file) {
```
Comment (dxuehu) [inline client/js/utils/fileTreeUtils.js]: refactor to one foreach

File: client/js/views/visualize.react.jsx (hunk 1 above)
Comment (dxuehu) [inline client/js/views/visualize.react.jsx]: loading should not be in state. it is derived from commits.length > 0

**Pull request merged**